### ISSUE TITLE
Plotly legend overlaps with barplot

### ISSUE THREAD

**Issue by luechtian** [jmbuhr/ShinyLipids#60]
![image](https://user-images.githubusercontent.com/51409126/91329637-abcc7480-e7c8-11ea-86d0-a7da71b391ab.png)

Next zoomlevel looks like this:
![image](https://user-images.githubusercontent.com/51409126/91330079-4036d700-e7c9-11ea-9310-b00d87d2c04b.png)

**Comment by jmbuhr** [comment]
I agree. This is why I definitely want to keep the underlying ggplot for the download buttons. The ggplot object gets passed to `plotly::ggplotly()` to convert it to a plotly plot (with it's own drawbacks, see previous commits where I had to change the ggplot) and then rendered. I previously played around with passing the plotly plot through some plotly layout functions but wasn't happy with the result so far, see the removed code in: https://github.com/jmbuhr/ShinyLipids/commit/232b9decdf4ed11e0be44a41e9d0be68a289d2b9.
The plotly plot is also not very manageable with smaller window sizes.
If you are up for the challenge, you could clone the repo, experiment around with the [plotly layout](https://plotly.com/r/reference/layout/) parameters and create a pull request. This would be really cool!

**Comment by luechtian** [comment]
Sure, why not :D  I will dig in to this!
